torchrl/data/tensordict/tensordict.py

```diff
@@ -2104,6 +2104,25 @@ def stack(
     return out
 
 
+def pad(tensordict: _TensorDict, pad_size: Sequence[int], value: float = 0.0):
```

Comment:
Do we have docstrings here, I could add explanation for the args.  I think allowing fewer dimensions makes sense since it would match the behavior of https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html (though adding left to right in dimensions rather than right to left).  A greater number returning an error makes sense

Reply:
Yeah if you want to sketch docstrings for pad it'd be awesome!
I would prefer if padding was going from left to right indeed, it's kind of the pattern of tensordict. If the tensordict is transparent about it I guess ppl will be fine with it (doc is everything when making weird decisions 🙃 )